.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# Default
+*    @tillkruss
+
+# Redis commands (Stack included)
+src/Command/Redis/*              @vladvildanov
```

Comment:
We should use @predis/team here instead of you @vladvildanov

Reply:
@chayim Right, fixed!